src/energyplus/ForwardTranslator.cpp

```diff
@@ -4521,11 +4534,6 @@ void ForwardTranslator::createStandardOutputRequests()
     {
       tableStyle.setString(OutputControl_Table_StyleFields::UnitConversion,"InchPound");
     }
-
-    IdfObject outputTableSummaryReport(IddObjectType::Output_Table_SummaryReports);
-    IdfExtensibleGroup eg = outputTableSummaryReport.pushExtensibleGroup();
-    eg.setString(Output_Table_SummaryReportsExtensibleFields::ReportName,"AllSummary");
-    m_idfObjects.push_back(outputTableSummaryReport);
```

Comment:
This will break backward compatibility in its current form. cf comment on line 558